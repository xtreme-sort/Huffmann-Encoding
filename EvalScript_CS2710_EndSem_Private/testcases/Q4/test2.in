GenerateLengthLimitedHuffman
ynxgglrxmsyruywkibxtdjeuueuapcdnrepkwofumfwpxkymanwvlcgkzycqveccihmguynnrmjdutaxcszhmtehnopknfbsssumgaahonnbaevkapkjulpjtykzogoahxhfjghfuhqybdfnngxcxwcbcvitbnsomrgumgbkhragsaktihjxvlershiufazleaxjqqjqmzoarhsuwiyuxkhbsntsewxmqnnclamvvxfxihtplbwwszayggnmdccaofpqzwhwbhdiqcfgkwsiyhxfintrrzuroiieggbgvjnjykasotkrkftpmzmizefedjvvtamejoufjqjivwmujvyflkmiktvrkzcpkjcsxxoutivwuanvncjxpvjttyesjyjcieohdjtgbjomxsjvwpvqxtuhyymdzhzhmyosutczjnbdhobbianaoaycsfomdtzkkraparbgiyuwqtogewxzsbhphzmnobvrsmzhuqagwuyytegmtvaqlokbppqwfpehtxqwzywaqfoveseykknypgedvalgkpvkqacpssytsyezpeqsguwccrqxsmvndryatgywtwwfwdukjzckrcdfeembkdekzrfxyygzhnbfwjjkrmobscaxtxlizgcay
9
Quit
