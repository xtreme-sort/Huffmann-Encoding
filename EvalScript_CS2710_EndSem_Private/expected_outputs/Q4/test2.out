a 0000
g 0001
k 0010
m 0011
s 0100
t 0101
y 0110
b 01110
c 01111
e 10000
f 10001
h 10010
i 10011
j 10100
n 10101
o 10110
p 10111
q 11000
r 11001
u 11010
v 11011
w 11100
x 11101
z 11110
d 111110
l 111111
cost: 3025
